.ci/set_kata_config.sh

```diff
@@ -35,18 +46,30 @@ Remove it if you want to use the stateless options.
 EOT
 }
 
-case ${request} in
-	true)
-		;;
-	false)
-		;;
-	*)
-		usage
-		exit 1
-		;;
-esac
-
-current_value=$(kata-runtime kata-env --json | jq ".Runtime.SandboxCgroupOnly")
+if [[ -z "${option}" || ! ${!valid_requests[*]} =~ ${option} ]]
+then
+	# unknown option
```

Comment:
Why not make this self-documenting?

```bash
echo >&2 "ERROR: unknown option: '$option'"
usage
exit 1
```

Same comment for unknown request below.

Reply:
@jodh-intel PTAL again :) maybe this can be simplified further